### DISCUSSION TITLE
CourseManager - backup via steam to different devices

### DISCUSSION THREAD

**Discussion by Torqueitup** [Courseplay/Courseplay_FS22#2866] [Q&A]
I use Courseplay on both my laptop and my desktop while playing FS22.
I've noticed that if CP routes are made on my laptop for example using CP's Coursemanager, they are not saved via Steams backup process so they can be downloaded and used on the same game on my Desktop computer.
Apart from manually transferring the relevant file, is there a workaround via the Steam process?

**Comment by Tensuko** [comment]
There is no workarround.
Steam only back up non mod related stuff.
